packages/scoreboard/pubspec.yaml

```diff
@@ -22,6 +22,7 @@ dependencies:
   dio: ^4.0.6
   intl: ^0.18.0
   dropdown_textfield: ^1.0.7
+  font_awesome_flutter: ^10.3.0
```

Comment:
What are we using `font_awesome_flutter` for?  @Yash-jar

Reply:
removed it